yast/src/include/smt/dialogs.rb

```diff
@@ -1196,7 +1196,7 @@ def ValidateCredentialsDialog(id, event)
       if nuuser == nil || nuuser == ""
         UI.SetFocus(Id("NUUser"))
         # Pop-up error message
-        Report.Error(_("NU User must not be empty."))
+        Report.Error(_("Updatge server user must not be empty."))
```

Comment:
Will fix the typo. Long-term, we should move towards SCC terms, but last time I checked even SCC was still somewhere between the two terminologies.